packages/webapp/src/shared/components/layout/header/header.component.tsx

```diff
@@ -74,34 +74,43 @@ export const Header = (props: HeaderProps) => {
             <Notifications templates={notificationTemplates} />
 
             <ProfileActions>
-              <Avatar
-                onClick={userDropdown.toggle}
-                tabIndex={0}
-                aria-expanded={userDropdown.isOpen}
-                aria-label={intl.formatMessage({
-                  id: 'Header / Open profile menu aria label',
-                  defaultMessage: 'Open profile menu',
-                })}
-              />
+              <Popover
+                open={userDropdown.isOpen}
+                onOpenChange={(open) => {
+                  userDropdown.setIsOpen(open);
+                }}
+              >
+                <PopoverTrigger>
+                  <Avatar
+                    onClick={userDropdown.toggle}
+                    tabIndex={0}
+                    aria-expanded={userDropdown.isOpen}
+                    aria-label={intl.formatMessage({
+                      id: 'Header / Open profile menu aria label',
+                      defaultMessage: 'Open profile menu',
+                    })}
+                  />
+                </PopoverTrigger>
 
-              <ClickAwayListener onClickAway={userDropdown.clickAway}>
```

Comment:
Can you check if `ClickAwayListener` is used anywhere else? If not we can remove this component.